src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,127 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIAL_LENGTH = 10;
+    private int totalCount;
+    private int filledCount;
+    private Object[] items;
```

Comment:
```suggestion
    private T[] items;
```
that's where you should use your type parameter